client/wfnews-war/src/main/angular/src/app/components/incident-identify-panel/incident-identify-panel.component.scss

```diff
@@ -53,6 +53,9 @@
     .label {
       color: #036;
     }
+    .paragraph-format{
```

Comment:
We already have a paragraph-format class with the same settings. You might be able to get this from a parent scss without the addition here